src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -261,4 +261,13 @@
   <data name="ResolutionSuccessfulNoAction" xml:space="preserve">
     <value>Resolution was successful but resulted in no action</value>
   </data>
+  <data name="ExceptionWhenTryingToAddSource" xml:space="preserve">
+    <value>Exception '{0}' thrown when trying to add source '{1}'. Please ensure all your online package sources are available.</value>
```

Comment:
`ensure` can be interpreted as if the user can somehow fix/control it, which may or may not be the case if the issue is server-side. Perhaps `verify` is more suitable?